README.md

```diff
@@ -136,6 +136,7 @@ OPTIONS:
    --dest-delete-extraneous-files, -d   Delete extraneous files on the destination by using rsync's '--delete' flag (default: false)
    --ignore-mounted, -i                 Do not fail if the source or destination PVC is mounted (default: false)
    --no-chown, -o                       Omit chown on rsync (default: false)
+   --read-only-mount, -R                Mount the PVC in ReadOnly mode (default: true)
```

Comment:
To be consistent with the other flags (sourceordest-verb-noun), can you apply the following suggestion? Also if you move it just below `--source-namespace` in README that'd be great.

```suggestion
   --source-mount-read-only, -R                Mount the source PVC in read-only mode (default: true)
```

And rename the variables respectively.

Reply:
all done!